homeassistant/generated/integrations.json

```diff
@@ -4867,6 +4867,12 @@
       "integration_type": "hub",
       "config_flow": false
     },
+    "playstation_network": {
```

Comment:
There are two entries for `playstation_network` in this file. I think you should remove this one and run `python3 -m script.hassfest`.